src/main/java/uk/gov/hmcts/darts/common/entity/TranscriptionCommentEntity.java

```diff
@@ -28,6 +28,9 @@ public class TranscriptionCommentEntity extends CreatedModifiedBaseEntity {
     @SequenceGenerator(name = "trc_gen", sequenceName = "trc_seq", allocationSize = 1)
     private Integer id;
 
+    @Column(name = "trw_id")
+    private Integer transcriptionWorkFlowId;
```

Comment:
This field should probably be renamed to have `Workflow` one word

Reply:
ACCEPT